feat: add backendBpparam,MsBackendSql

jorainer opened the pull request:
- Add the `backendBpparam` method for `MsBackendSql` hence ensuring parallel processing is disabled for `Spectra` objects using that backend.

sgibb commented:
@jorainer Thanks! What about
> Currently it doesn't check the type of the *user defined* columns. Wouldn't that be useful as well?

sgibb commented:
Right, would require the user to specify a named argument `columns = c(msLevel = "integer")`. But we could discuss this on different place. I am merging this PR now.

lgatto commented:
Great, thanks!

jorainer committed:
feat: add backendBpparam,MsBackendSql

- Add the `backendBpparam` method for `MsBackendSql` hence ensuring parallel
  processing is disabled for `Spectra` objects using that backend.
